java-1.js

```diff
@@ -0,0 +1,22 @@
+const myopen = document.querySelector('.open');
+const nav = document.querySelector('.nav-links-2');
+const myaxe = document.querySelector('.axe');
+const mytra1 = document.querySelector('.nav-transition-1');
+const mytra2 = document.querySelector('.nav-transition-2');
+const mytra3 = document.querySelector('.nav-transition-3');
+
+myopen.onclick = function () {
+  nav.style.cssText = 'transform:translateY(0%);';
+};
```

Comment:
[OPTIONAL] Nice work 👍🏽 but could be cool to use an arrow function, it would remove the warning ⚠️ you're getting and this would become 👇🏽 
```
myopen.onclick =  () =>  {
  nav.style.cssText = 'transform:translateY(0%);';
};
```
You can follow the same principle for the other functions. 🤓